### PR TITLE
Feature/#304 design notification handler for rejections

### PR DESCRIPTION
**Pull request by HenkKodde**
UML Map Reject notification as per story #304.

### CODE REVIEW

**Comment by elnyry** [comment]
ok, here are some changes
1. Please use dotted lines for the callbacks/responses (currently steps 4, 5)
2. Please add “Error code: 2001” for steps 3 and 4

File: CentralServices/seq_diagrams/seq-notification-reject-5.1.1.plantumi [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+box "Central Service" #LightYellow
+    participant CSAPI
+    participant TOPIC_NOTIFICATIONS
+end box
+
+' start flow
+activate NOTIFY_HANDLER
+
+group DFSP Notified of Rejection
+    NOTIFY_HANDLER -> TOPIC_NOTIFICATIONS: Consume Notification event message \n <color #FF0000><b>Error code:</b> 2001 </color>
```
Comment (mdebarros) [inline CentralServices/seq_diagrams/seq-notification-reject-5.1.1.plantumi]: Missing Event log upon consuming the notification. Need to reference seq-event-9.1.0 (which has yet to be designed).

File: CentralServices/seq_diagrams/seq-notification-reject-5.1.1.plantumi [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            {
+                "errorInformation": {
+                    "errorCode": <errorCode>,
+                    "errorDescription": <ErrorMessage>,
+                }
+            }
+        end note
+
+        note left of NOTIFY_HANDLER #lightblue
+            Log ERROR Messages
```
Comment (mdebarros) [inline CentralServices/seq_diagrams/seq-notification-reject-5.1.1.plantumi]: Need to reference seq-event-9.1.0 (which has yet to be designed).

**Comment by elnyry** [comment]
@mdebarros - about this comment "Should this not cater for timeout rejections (https://github.com/mojaloop/docs/blob/develop/CentralServices/seq_diagrams/seq-timeout-2.3.3.svg)?"

Yes, it should - @HenkKodde

**Comment by HenkKodde** [comment]
Changes was done as requested. Meeting with Miguel and Nico confirmed they are comfortable with the updates and ready for final review.

**Comment by HenkKodde** [comment]
Thank you - merging and closing the PR and story.

**Pull request merged**